pages/common/stack.md

```diff
@@ -0,0 +1,31 @@
+# stack
+
+> Tool for managing haskell projects.
+
+- Create a new project:
+
+`stack new {{project_name}}`
+
+- Install all packages needed by a project:
+
+`stack install`
+
+- Compile a project:
+
+`stack build`
+
+- Run tests inside a project:
+
+`stack test`
+
+- Compile a project and re-compile every time a file changes:
+
+`stack build --file-watch`
+
+- Compile a project and execute a command after compilation:
+
+`stack build --exec "{{command}}"`
+
+- Compile a project and re-compile every time a file changes and execute a command after compilation:
```

Comment:
I'm not sure this one is needed since you've already demonstrated each function individually above.

Reply:
I wanted to demonstrate that you can mix these parameters as you want.
I will remove it :+1:

Reply:
I think it's something that the average user should be able to figure out :-)

@agnivade / @waldyrious thoughts?

Reply:
Less is good. But not at the cost of clarity. In this case, I think it's fine.

Reply:
Does that mean I should re-add this example?

Reply:
Nope. It's fine. i.e. keep it removed.